JSQMessagesViewController/Controllers/JSQMessagesViewController.m

```diff
@@ -724,6 +732,13 @@ - (void)textViewDidChange:(UITextView *)textView
         return;
     }
 
+    if (!textView.text.length) {
+        //workaroung for textview change back to default
```

Comment:
After pasting an attributed text the input text box text format remained the same as pasted text and did not change back to how it was before. see bellow picture after pasting and sending an attributed message with large font:
![screen shot 2015-12-07 at 10 38 46 am](https://cloud.githubusercontent.com/assets/9285802/11631213/f413c54e-9cce-11e5-9874-52dbc9753195.png)

Is should be like this:
![screen shot 2015-12-07 at 10 41 13 am](https://cloud.githubusercontent.com/assets/9285802/11631234/12d9857c-9ccf-11e5-8403-a7b1ba20bec6.png)